rllib/BUILD

```diff
@@ -592,6 +592,14 @@ py_test(
     data = ["tests/data/cartpole/large.json"],
     srcs = ["algorithms/bc/tests/test_bc.py"]
 )
+py_test(
```

Comment:
nit: remove "_new_stack"

From here on "new stack" is the norm. Rather add `_old_api_stack` to all other tests to distinguish.